Update README.md (#13)
opsdroid/connector-shell: README.md

> ⚠️ *DEPRECATED* ⚠️ This connector is now built in to [opsdroid core](https://opsdroid.readthedocs.io/en/stable/connectors/slack/). This repository only exists for backward compatibility and will be removed.

Reviewer: You're linking to the slack connector but it should be the shell connector. I

```suggestion
⚠️ *DEPRECATED* ⚠️ This connector is now built into [opsdroid core](https://docs.opsdroid.dev/en/stable/connectors/shell/). This repository only exists for backward compatibility and will be removed.
```

Reviewer: Ah you beat me to it!

Author: should i take the readthedocks.io or opsdroid.dev link?

Reviewer: I would go with the docs.opsdroid.dev, it's relatively new.
